contributors/devel/sig-architecture/api_changes.md

```diff
@@ -1016,6 +1026,155 @@ In future Kubernetes versions:
     }
     ```
 
+#### New enum value in existing field
+
+A developer is considering adding a new allowed enum value of `"OnlyOnTuesday"`
+to the following existing enum field:
+
+```go
+type Frobber struct {
+  // restartPolicy may be set to "Always" or "Never".
+  // Additional policies may be defined in the future.
+  // Unrecognized policies should be treated as "Never".
+  RestartPolicy string `json:"policy"
+}
+```
+
+Older versions of expected API clients must be able handle the new value in a safe way:
+
+* If the enum field drives behavior of a single component, ensure all versions of that component
+  that will encounter API objects containing the new value handle it properly or fail safe.
+  For example, a new allowed value in a `Pod` enum field consumed by the kubelet must be handled
+  safely by kubelets up to two versions older than the first API server release that allowed the new value.
+* If an API drives behavior that is implemented by external clients (like `Ingress` or `NetworkPolicy`),
+  the enum field must explicitly indicate that additional values may be allowed in the future,
+  and define how unrecognized values must be handled by clients. If this was not done in the first release
```

Comment:
> If this was not done in the first release

We know lots of such fields DO NOT document the assumed default.  Are they frozen forever?  Do we have an exit path for them?

Reply:
> We know lots of such fields DO NOT document the assumed default. Are they frozen forever?

Requiring explicit documentation for open-ended enums was already stated above:

> Adding a new value to an enumerated set is _not_ a compatible change. [...] For enumeration-like fields that expect to add new values in the future, such as reason fields, please document that expectation clearly in the API field descriptions. Clients should treat such sets of values as potentially open-ended.

If there was no documentation that future values might be added, AND the field drives behavior in unknown external clients, I don't see how new values can be safely added.

I strengthened the requirement above and added the bit about defining how clients should treat unknown values.